src/views/htmlcontent/src/js/slick.dragrowselector.js

```diff
@@ -116,10 +116,16 @@
 
                 if (e.shiftKey) {
                     var last = _ranges.pop();
+
+                    var toCell = Math.max(newRangeColumn, last.toCell);
+                    if(last.fromCell <= newRangeColumn) {
```

Comment:
It is possible to add eslint to our repo as well if desired to lint the js files, but i think there are only 2 currently.